packages/node_modules/@node-red/editor-client/locales/de/editor.json

```diff
@@ -508,8 +543,8 @@
             "advanced": "Fortgeschritten"
         },
         "actions": {
-            "collapse-all": "Kategorien einklappen",
-            "expand-all": "Kategorien ausklappen"
+            "collapse-all": "Alle Kategorien einklappen",
+            "expand-all": "Alle Kategorien ausklappen"
```

Comment:
Could you tell us why you edited these value please?
Do they align better with the English version?
Did you check the UI after changing to ensure they dont overflow or push any elements around?

Reply:
Keep as is. There's no added-value in explicitly stating "all" ... as there's no option to just collapse 'a few'. It's fair yet to highlight that the english version states "collapse all categories".